plugins/elasticsearch/loggable-transport-netty4/build.gradle

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
Also flagging this as well -- this looks really wrong to me.  It's weird that it says "licensed to Elasticsearch"

Reply:
This is not needed for new code.